lxd/daemon.go

```diff
@@ -754,6 +754,9 @@ func (d *Daemon) createCmd(restAPI *mux.Router, version string, c APIEndpoint) {
 			return
```

Comment:
Suggested change for commit message: `lxd: Set OpenFGA cache in request context for all requests`.

Reply:
fixed